lilypad_module.json.tmpl

```diff
@@ -12,10 +12,10 @@
       },
       "Docker": {
         "Entrypoint": [
-          "/usr/games/cowsay",
+          "/usr/local/bin/cowsay",
           {{if .Message}}{{.Message}}{{else}}"Pass me an input called Message, like lilypad run cowsay:v0.0.2 -i Message=moo"{{end}}
```

Comment:
Let's update the default text here to something more generic (as it will be out of date every time for certain):
`"Pass an input called Message: -i Message=moo"`

Reply:
that's a great catch thanks, I push a change to fix it